numpy/core/src/umath/override.c

```diff
@@ -548,15 +554,13 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
             goto fail;
         }
 
-        /* Access the override */
-        array_ufunc = PyObject_GetAttrString(override_obj,
-                                             "__array_ufunc__");
-        if (array_ufunc == NULL) {
-            goto fail;
-        }
+        /* Set the self argument, since we have an unbound method */
+        Py_INCREF(override_obj);
+        PyTuple_SetItem(override_args, 0, override_obj);
```

Comment:
Because `PyArray_LookupSpecial` returns an unbound method. Arguably it should return a bound one, but that's wasteful to construct, and I wasn't able to make it work